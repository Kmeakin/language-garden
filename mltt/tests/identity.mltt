let id : fun (A : Type) -> A -> A :=
  fun A := fun a := a;

id { A : Type } { A := {} }
