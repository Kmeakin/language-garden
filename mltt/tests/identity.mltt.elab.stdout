<input> : { A : Type; } :=
  let _ : fun (_ : Type) -> fun (_ : @0) -> @1 := fun (_ : Type) => fun
                                                                    (_ : @0)
                                                                    =>
                                                                    @0;
    @0
  {
    A : Type;
  }
  {
    A := {};
  }
